packages/react-dom/src/client/__tests__/setInnerHTML-test.js

```diff
@@ -9,60 +9,61 @@
 
 'use strict';
 
-// TODO: can we express this test with only public API?
-var setInnerHTML = require('../setInnerHTML');
-var Namespaces = require('../../shared/DOMNamespaces').Namespaces;
+const React = require('react');
+const ReactDOM = require('react-dom');
 
-describe('setInnerHTML', () => {
```

Comment:
Let's also rename test file?

Reply:
Ah yes, I missed that. Cool if I rename it here and also the file itself?

Reply:
On it!